x/evm/keeper/balance.go

```diff
@@ -0,0 +1,37 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/sei-protocol/sei-chain/x/evm/types"
+)
+
+func (k *Keeper) GetBalance(ctx sdk.Context, address common.Address) uint64 {
+	store := ctx.KVStore(k.storeKey)
+	bz := store.Get(types.BalanceKey(address))
+	if bz == nil {
+		return 0
+	}
+	balance := sdk.Coin{}
+	if err := balance.Unmarshal(bz); err != nil {
+		// this should never happen
+		panic(err)
+	}
+	return balance.Amount.Uint64()
+}
+
+func (k *Keeper) SetBalance(ctx sdk.Context, address common.Address, amount uint64) {
+	store := ctx.KVStore(k.storeKey)
+	if amount == 0 {
```

Comment:
would be good to note that this has a side effect if balance == 0 (comment or function name)

Reply:
yeah updated function name